lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeedFormatter.cs

```diff
@@ -48,20 +43,20 @@ public SyndicationFeed Feed
 
         public abstract bool CanRead(XmlReader reader);
 
-        public abstract void ReadFrom(XmlReader reader);
+        public abstract Task ReadFromAsync(XmlReader reader);
 
```

Comment:
We should have a CancellationToken parameter on all the public async methods. Even if we don't initially do anything with the token, there are multiple ways to cancel the operation (exact implementation can be decided later) but we need the parameter in the method at the start.